statistics/update_test.go

```diff
@@ -148,18 +148,18 @@ func (s *testStatsUpdateSuite) TestMultiSession(c *C) {
 		testKit2.MustExec("delete from test.t1 limit 1")
 	}
 
-	err = testKit.Se.Close()
```

Comment:
Session should not be used anymore, if it's Closed !!!
Close means all resource bind to it should be released, if you violate the constraint, that's really bad design.